src/index.css

```diff
@@ -263,4 +265,22 @@ html::-webkit-scrollbar-track {
 }
 .Toastify{
   font-size: 16px;
```

Comment:
you can use rem values

Reply:
actually that's a static size don't need that's why did so there